tests/test_range.py

```diff
@@ -2,10 +2,13 @@
 Unit-tests for DateRange.
 """
 
-import unittest
```

Comment:
If you wish to make styling changes, that's fine, but please submit that as a separate PR.